test/docker-e2e/e2e_test.go

```diff
@@ -22,7 +23,22 @@ import (
 const (
 	txsimImage = "ghcr.io/celestiaorg/txsim"
 	txSimTag   = "v4.0.10-arabica"
-	homeDir    = "/var/cosmos-chain/celestia"
+
+	// Liveness check constants
+	defaultBlocksPerValidator = 3                    // Minimum blocks each validator should propose for liveness validation
+	validatorAddrTruncateLen  = 8                    // Length to truncate validator addresses for display
+	validatorNameFormat       = "validator_%s"       // Format string for validator display names
+	nodeStatusFormat          = "node_%d(height_%d)" // Format string for node status messages
```

Comment:
[question] these format strings seem like a code smell. If they're used in format strings then can they be inlined? Without more context, I don't know what the %s or %d mean in these logs so it's more confusing to separate this from the actual usage.